smithy-aws-protocol-tests/model/restXml/services/s3.smithy

```diff
@@ -515,10 +551,14 @@ string AccountId
 
 string BucketName
 
+string KeyName
+
 string Delimiter
 
 string DisplayName
 
+string Expires
```

Comment:
Should this be a timestamp? From what I understand, clients should try deserializing as a timestamp, but not fail Expires is invalid, right?